src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,29 @@
 package qna.domain;
 
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class AnswerTest {
     public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
     public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+    
+    @Test
+    void Answer_contents_test() {
```

Comment:
```suggestion
    void answer_contents_test() {
```
메서드이므로 소문자로 작성해주면 좋을 것 같습니다. 추가로, `DisplayName으로 어떤 테스트인지 구체적으로 작성해주어도 좋을 것 같아요...!